src/components/Menu/RepoActions/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex } from '@aws-amplify/ui-react';
+import ExternalLink from '../../ExternalLink';
+import React from 'react';
+import { IconPencil } from '@/components/Icons';
+
+function createEditLink(directoryPath) {
+  const safePath = directoryPath
+    .split('/')
+    .map(encodeURIComponent)
+    .join('/');
+  return `https://github.com/aws-amplify/docs/edit/main/src/pages${safePath}/`;
+}
+
+export default function RepoActions({ path }) {
+  console.log(path.path);
```

Comment:
nit: Just a reminder to remove these console statements before you're ready to merge.